code/modules/projectiles/pins.dm

```diff
@@ -222,4 +222,25 @@ Pins Below.
 
 /obj/item/device/firing_pin/away_site/pin_auth(mob/living/user)
 	var/turf/T = get_turf(src)
-	return !isStationLevel(T.z)
\ No newline at end of file
+	return !isStationLevel(T.z)
+
+/obj/item/device/firing_pin/security_level
+	name = "security level firing pin"
+	desc = "This security level locked firing pin allows weapons to be fired only when the security level is elevated."
+	fail_message = "<span class='warning'>SECURITY LEVEL INSUFFICIENT.</span>"
```

Comment:
SPAN macro

Reply:
Sorry? I don't quite understand. I made it identical to all the other firing pins.

Reply:
Search up 'SPAN_' and look at the defines there. 

Stuff like:
"<span class='warning'>words</span>"

Should instead be written like:
SPAN_WARNING("words")

Reply:
A *lot* of older stuff does not do this, but one day we're going to have to change them all to the SPAN macros, so it's easier for us all if people do it as we go along.

Reply:
Okay in this case it's because fail_message has to be static. Build fails when you use the span macro. This is something that could be fixed, but requires someone to overhaul pins.dm which is outside the scope of this PR.

Reply:
Oh, that's fine then.